scripts/install

```diff
@@ -183,7 +190,7 @@ ynh_replace_string "__APPNAME__" "$appname" "$finalnginxconf"
 
 # Set permissions to app files
 chmod 775 -R $final_path
-chown -R $admin:$appname $final_path
+chown -R $appname:$appname $final_path
```

Comment:
To set the permission to admin was on purpose to allow the admin to use the directory and copy his files.